web/libs/datamanager/src/components/Common/Modal/Modal.scss

```
@@ -0,0 +1,128 @@
+.modal {
+  --transition-duration: 100ms;
+
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 10000;
+  display: flex;
+  position: absolute;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(#000, 0.3);
```

Flagging as opportunity for css var replacement. Further review of usage needed.
```suggestion
  background-color: rgba(var(--black), 0.3);
```